actors/util/adt/balancetable.go

```diff
@@ -40,6 +41,12 @@ func (t *BalanceTable) Get(key addr.Address) (abi.TokenAmount, error) {
 	return value, nil
 }
 
+// Has checks if the balance for a key exists
```

Comment:
This changes the prior invariants on this type. Assuming it's a necessary change, please remove the comment at line 15.